<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormUsbExport</class>
 <widget class="QWidget" name="FormUsbExport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>845</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(85, 87, 83);
color: rgb(238, 238, 236);</string>
     </property>
     <property name="text">
      <string>Receitas no Sistema:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="lst_local">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="spacing">
      <number>16</number>
     </property>
     <property name="viewMode">
      <enum>QListView::ListMode</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(211, 215, 207);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="ButtonBeR" name="btn_to_usb_all">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>110</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>110</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>EXPORTAR todas as
receitas SISTEMA -&gt; USB</string>
        </property>
        <property name="icon">
         <iconset resource="../res/icons.qrc">
          <normaloff>:/icons/recipe_export.png</normaloff>:/icons/recipe_export.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>32</height>
         </size>
        </property>
        <property name="bgIcon">
         <iconset resource="../res/icons.qrc">
          <normaloff>:/icons/bt-blank.png</normaloff>
          <normalon>:/icons/bt-blank_pressed.png</normalon>:/icons/bt-blank.png</iconset>
        </property>
        <property name="bgIconSize">
         <size>
          <width>80</width>
          <height>80</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="ButtonBeR" name="btn_from_usb">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>110</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>110</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>IMPORTAR receita
USB -&gt; SISTEMA</string>
        </property>
        <property name="icon">
         <iconset resource="../res/icons.qrc">
          <normaloff>:/icons/recipe_import.png</normaloff>:/icons/recipe_import.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>32</height>
         </size>
        </property>
        <property name="bgIcon">
         <iconset resource="../res/icons.qrc">
          <normaloff>:/icons/bt-blank.png</normaloff>
          <normalon>:/icons/bt-blank_pressed.png</normalon>:/icons/bt-blank.png</iconset>
        </property>
        <property name="bgIconSize">
         <size>
          <width>80</width>
          <height>80</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_9">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="ButtonBeR" name="btn_to_usb">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>110</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>110</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>EXPORTAR receita
SISTEMA -&gt; USB</string>
        </property>
        <property name="icon">
         <iconset resource="../res/icons.qrc">
          <normaloff>:/icons/recipe_export.png</normaloff>:/icons/recipe_export.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>32</height>
         </size>
        </property>
        <property name="bgIcon">
         <iconset resource="../res/icons.qrc">
          <normaloff>:/icons/bt-blank.png</normaloff>
          <normalon>:/icons/bt-blank_pressed.png</normalon>:/icons/bt-blank.png</iconset>
        </property>
        <property name="bgIconSize">
         <size>
          <width>80</width>
          <height>80</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="ButtonBeR" name="btn_config_from_usb">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>110</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>110</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>IMPORTAR configurações
USB -&gt; SISTEMA</string>
        </property>
        <property name="icon">
         <iconset resource="../res/icons.qrc">
          <normaloff>:/icons/config_import.png</normaloff>:/icons/config_import.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="bgIcon">
         <iconset resource="../res/icons.qrc">
          <normaloff>:/icons/bt-blank.png</normaloff>
          <normalon>:/icons/bt-blank_pressed.png</normalon>:/icons/bt-blank.png</iconset>
        </property>
        <property name="bgIconSize">
         <size>
          <width>80</width>
          <height>80</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="ButtonBeR" name="btn_config_to_usb">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>110</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>110</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>EXPORTAR configurações
SISTEMA -&gt; USB</string>
        </property>
        <property name="icon">
         <iconset resource="../res/icons.qrc">
          <normaloff>:/icons/config_export.png</normaloff>:/icons/config_export.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="bgIcon">
         <iconset resource="../res/icons.qrc">
          <normaloff>:/icons/bt-blank.png</normaloff>
          <normalon>:/icons/bt-blank_pressed.png</normalon>:/icons/bt-blank.png</iconset>
        </property>
        <property name="bgIconSize">
         <size>
          <width>80</width>
          <height>80</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_10">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(85, 87, 83);
color: rgb(238, 238, 236);</string>
     </property>
     <property name="text">
      <string>Receitas no drive USB:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="lst_usb">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="spacing">
      <number>16</number>
     </property>
     <property name="viewMode">
      <enum>QListView::ListMode</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1">
     <item>
      <widget class="QLabel" name="lbl_usb_device">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>55</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>-</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="ButtonBeR" name="btn_return">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/icons/arrow_backward.png</normaloff>:/icons/arrow_backward.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="bgIcon">
        <iconset resource="../res/icons.qrc">
         <normaloff>:/icons/bt-blank.png</normaloff>
         <normalon>:/icons/bt-blank_pressed.png</normalon>:/icons/bt-blank.png</iconset>
       </property>
       <property name="bgIconSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ButtonBeR</class>
   <extends>QPushButton</extends>
   <header>buttonber.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res/icons.qrc"/>
 </resources>
 <connections/>
</ui>
